checks/webhook.go

```diff
@@ -38,28 +38,23 @@ func init() {
 
 // WebHooks run Webhooks check.
 func WebHooks(c *checker.CheckRequest) checker.CheckResult {
-	// TODO: remove this check when v6 is released
-	_, enabled := os.LookupEnv("SCORECARD_EXPERIMENTAL")
```

Comment:
I don't think we should remove this as part of this conversion PR. Any enabling should be explicit, and not hidden in this PR.